src/NuGet.Core/NuGet.Packaging/PackagesConfig.cs

```diff
@@ -38,14 +38,17 @@ public static bool TryGetAttribute(XElement node, string name, out string value)
         /// </summary>
         public static bool HasAttributeValue(XElement node, string attributeName, string targetValue, out XElement element)
         {
-            foreach (var package in node.Elements(XName.Get(PackageNodeName)))
```

Comment:
when can the node be null? That seems like a case where we should throw and not return something